lgc/builder/DescBuilder.cpp

```diff
@@ -80,6 +80,8 @@ Value *DescBuilder::CreateLoadBufferDesc(unsigned descSet, unsigned binding, Val
 
     std::tie(topNode, node) = m_pipelineState->findResourceNode(abstractType, descSet, binding);
     if (!node) {
+      if (!m_pipelineState->isUnlinked())
+        getPipelineState()->setError("Resource node is not found!");
```

Comment:
I guess you mean this to be an unrecoverable error, not something where the front-end would retry in a different way. If so, it should call report_fatal_error().

Reply:
It is also quite common to just use an assertion.